load_confounds.py

```diff
@@ -290,30 +216,139 @@ def load_confounds(
+    Ciric et al., 2017 "Benchmarking of participant-level confound regression
+    strategies for the control of motion artifact in studies of functional
+    connectivity" Neuroimage 154: 174-87
+    https://doi.org/10.1016/j.neuroimage.2017.03.020
+    """
+
+    def __init__(
+        self,
+        strategy=["motion", "high_pass", "wm_csf"],
+        motion="full",
+        n_motion=0,
+        wm_csf="basic",
+        global_signal="basic",
+        compcor="anat",
+        n_compcor=6,
+    ):
+        self.strategy = _sanitize_strategy(strategy)
+        self.motion = motion
+        self.n_motion = n_motion
+        self.wm_csf = wm_csf
+        self.global_signal = global_signal
+        self.compcor = compcor
+        self.n_compcor = n_compcor
+
+    def load(self, confounds_raw):
+        """
+        Load fMRIprep confounds
+
+        Parameters
+        ----------
+        confounds_raw : Pandas Dataframe or path to tsv file(s), optionally as a list.
+            Raw confounds from fmriprep
+
+        Returns
+        -------
+        confounds :  pandas DataFrame or list of pandas DataFrame
+            A reduced version of fMRIprep confounds based on selected strategy and flags.
+        """
+        confounds_raw, flag_single = _sanitize_confounds(confounds_raw)
+        confounds_out = []
+        for file in confounds_raw:
+            confounds_out.append(self._load_single(file))
+
+        # If a single input was provided,
+        # send back a single output instead of a list
+        if flag_single:
+            confounds_out = confounds_out[0]
+
+        self.confounds_ = confounds_out
+        return confounds_out
+
+    def _load_single(self, confounds_raw):
+        """Load a single confounds file from fmriprep."""
+        # Convert tsv file to pandas dataframe
+        confounds_raw = _confounds2df(confounds_raw)
+
+        confounds = pd.DataFrame()
+
+        if "motion" in self.strategy:
+            confounds_motion = _load_motion(confounds_raw, self.motion, self.n_motion)
+            confounds = pd.concat([confounds, confounds_motion], axis=1)
+
+        if "high_pass" in self.strategy:
+            confounds_high_pass = _load_high_pass(confounds_raw)
+            confounds = pd.concat([confounds, confounds_high_pass], axis=1)
+
+        if "wm_csf" in self.strategy:
+            confounds_wm_csf = _load_wm_csf(confounds_raw, self.wm_csf)
+            confounds = pd.concat([confounds, confounds_wm_csf], axis=1)
+
+        if "global" in self.strategy:
+            confounds_global_signal = _load_global(confounds_raw, self.global_signal)
+            confounds = pd.concat([confounds, confounds_global_signal], axis=1)
+
+        if "compcor" in self.strategy:
+            confounds_compcor = _load_compcor(
+                confounds_raw, self.compcor, self.n_compcor
+            )
+            confounds = pd.concat([confounds, confounds_compcor], axis=1)
+
+        return confounds
+
+class P2(Confounds):
```

Comment:
it's a bit of a tough call. I was trying to follow the names of Ciric et al. 2017, but wasn't able to follow closely anyway because it's `6P` and a Class name cannot start by a number. I think that while we are changing it, we could go for `params`. But because class names are capitalized in Python, it would be `Params6`. What do you think?

Reply:
Sounds good!